src/tests/sqlsmith/src/lib.rs

```diff
@@ -461,3 +463,29 @@ pub fn create_table_statement_to_table(statement: &Statement) -> Table {
         _ => panic!("Unexpected statement: {}", statement),
     }
 }
+
+/// Error workarounds
+
+fn is_division_by_zero_err(db_error: &str) -> bool {
+    db_error.contains(&ExprError::DivisionByZero.to_string())
+}
+
+fn is_numeric_out_of_range_err(db_error: &str) -> bool {
+    db_error.contains(&ExprError::NumericOutOfRange.to_string())
+}
+
+/// Workaround to permit runtime errors not being propagated through channels.
+/// FIXME: This also means some internal system errors won't be caught.
+/// Tracked by: <https://github.com/risingwavelabs/risingwave/issues/3908#issuecomment-1186782810>
+fn is_broken_chan_err(error: &str) -> bool {
```

Comment:
Issue is resolved, frontend should receive error now, see https://github.com/risingwavelabs/risingwave/issues/4811#issuecomment-1305043650. This should be removed